FunctorCategories/gap/Functors.gi

```diff
@@ -527,65 +527,69 @@ InstallMethod( EquivalenceFromFullSubcategoryOfProjectivesObjectsIntoAdditiveClo
     
 end );
 
-##
-BindGlobal( "SET_ISOMORPHISMS_BETWEEN_SOURCE_AND_IMAGE_OF_YONEDA_EMBEDDING_OF_SOURCE",
+if IsPackageMarkedForLoading( "ToolsForHigherHomologicalAlgebra", ">= 2023.03-01" ) then
```

Comment:
Is this enough? Don't you need `ComplexesCategories` in the code below? If yes, then please bump its version in the suggested dependency.

Reply:
Should be enough, these isomorphisms only require `AdditiveFunctorByTwoFunctions` which is implemented in `ToolsForHigherHomologicalAlgebra`.

Reply:
Then please add `ToolsForHigherHomologicalAlgebra` as a suggested package and put the version there and delete it here.